packages/by-name/kata/contrast-node-installer-image/package.nix

```diff
@@ -116,13 +120,17 @@ let
   kata-container-img = ociLayerTar {
     files = [
       {
-        source = kata.kata-image;
+        source = "${kata.kata-image}/${kata.kata-image.imageFile}";
```

Comment:
Could this be referenced directly as `${kata.kata-image.imageFile}`? It's a bit confusing that the path needs to be appended here.